runtime/onert/core/src/compiler/Fp32ToFp16Converter.cc

```diff
@@ -290,7 +294,11 @@ void Fp32ToFp16Converter::appendNewOpSeqForConvertFp16ToFp32(const ir::OpSequenc
 {
   // OpSeq's output set is included in the last operation's output set
   const ir::OperandIndexSequence op_seq_outputs = op_seq.getOutputs(); // copied
-  for (const auto &op_seq_output_ind : op_seq_outputs)
+
+  // Remove duplicated input
+  // NOTE Please do not change sequence of op_seq_outputs. It can change the sequence of outputs of
```

Comment:
What is the meaning of sequence after removing duplications?

Reply:
The for-loop below should keep sequence of `op_seq_inputs` after removing duplication. Operations's inputs sequence is also changed when I changed sequence of `op_seq_inputs`.
But I didn't confirm why. So it's a defensive comment. This commit does not change sequence of `op_seq_inputs` and `op_seq_outputs`

Reply:
I roughly guess it may be another issue in the existing code..